scss/_patterns_image.scss

```diff
@@ -0,0 +1,18 @@
+@import 'settings';
+
+@mixin vf-p-image {
+  .p-image {
+
+    &--bordered {
+      border: {
+        color: $color-mid-light;
+        style: solid;
+        width: 1px;
+      }
+    }
+
+    &--shadowed {
+      box-shadow: 0 1px 5px 1px transparentize(rgba($color-mid-light, .2)); // sass-lint:disable-line no-color-literals
```

Comment:
Transparentize needs two parameters. A colour (`$color-mid-light`) and a ratio to make transparent. The second works in reverse to how rgba works. Therefore, to achieve the same CSS you need to use `.8`.

https://www.sassmeister.com/gist/729f518a3c8bacfef71b64571c70f200

You can then remove the disable of the sass-lint.

Reply:
Didn't know this -thanks!